modules/mod_acl_user_groups/mod_acl_user_groups.erl

```diff
@@ -115,6 +115,7 @@ event(#postback{message={delete_all, Args}}, Context) ->
             z_render:wire({alert, [{message, ?__("Delete is canceled, there are users in the user groups.", Context)}]}, Context)
     end.
 
+-spec ug_delete(list(pos_integer()), #context{}) -> any().
```

Comment:
The spec in line 118 uses a record, please define a type for the record and use that instead.